packages/js/components/src/rich-text-editor/rich-text-editor.tsx

```diff
@@ -28,12 +28,14 @@ registerBlocks();
 
 type RichTextEditorProps = {
 	blocks: BlockInstance[];
+	label?: string;
```

Comment:
What about `className` for `has-error` values when the editor is invalid and `help` to render the error message during validations? If your are passing the `label` then the prev props should be passed also. Otherwise consider wrapping this component inside a [BaseControl](https://developer.wordpress.org/block-editor/reference-guides/components/base-control/).

Reply:
See [my above comment ](https://github.com/woocommerce/woocommerce/pull/34961#discussion_r991471662)on the `help` prop.  I think that prop needs more thought as this component does not match the patterns of most base control inputs.